Sources/Fuzzilli/FuzzIL/Operations.swift

```diff
@@ -240,6 +240,20 @@ class CreateArrayWithSpread: Operation {
     }
 }
 
+class CreateTemplateLiteral: Operation {
+    // Stores the string elements of the temaplate literal
+    let literals: [String]
+
+    var numIdentifiers: Int {
+        return numInputs
+    }
+
+    init(literals: [String]) {
+        self.literals = literals
```

Comment:
How about we enforce that `literals.count > 1` always? I think that would simplify some things. So just add an `assert(parts.count > 0)` here, then drop the check from the following line.